addon/components/nrg-sidebar.hbs

```diff
@@ -1,8 +1,7 @@
 <NrgModal
   @sidebar={{true}}
   @isOpen={{this.application.sidebarIsOpen}}
-  {{did-insert this.onInsert}}
-  {{will-destroy this.onDestroy}}
+  {{on-resize onResize}}
```

Comment:
`onResize` needs the `@action` decorator to preserve the context of `this`

Also:
```suggestion
  {{on-resize this.onResize}}
```